### PR TITLE
Bumping dependencies

### PR DESCRIPTION
**Pull request by thekp**
**Overall change:** _Bump 3rd party dependencies._

**Cover the following dependabot :**
https://github.com/bbc/simorgh/pull/4142, https://github.com/bbc/simorgh/pull/4141, https://github.com/bbc/simorgh/pull/4140, https://github.com/bbc/simorgh/pull/4139, https://github.com/bbc/simorgh/pull/4138, https://github.com/bbc/simorgh/pull/4137, https://github.com/bbc/simorgh/pull/4136, https://github.com/bbc/simorgh/pull/4132, https://github.com/bbc/simorgh/pull/4131, https://github.com/bbc/simorgh/pull/4130, https://github.com/bbc/simorgh/pull/4129, https://github.com/bbc/simorgh/pull/4128, https://github.com/bbc/simorgh/pull/4127, https://github.com/bbc/simorgh/pull/4126, https://github.com/bbc/simorgh/pull/4125, https://github.com/bbc/simorgh/pull/4124, https://github.com/bbc/simorgh/pull/4142

**Testing:**
Regression test

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -166,9 +166,9 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react-test-renderer": "^16.9.0",
+    "react-test-renderer": "^16.10.2",
     "start-server-webpack-plugin": "^2.2.5",
-    "storybook-chromatic": "^2.2.2",
+    "storybook-chromatic": "^3.0.0",
```
Comment (jroebu14) [inline package.json:172]: worth checking this still works

**Review by jroebu14 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -98,19 +98,19 @@
...
     "ramda": "^0.26.1",
-    "react": "^16.10.1",
-    "react-dom": "^16.10.1",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
     "react-helmet": "^5.2.1",
     "react-lazyload": "^2.6.2",
     "react-router-config": "^5.1.1",
-    "react-router-dom": "^5.0.1",
+    "react-router-dom": "5.0.1",
```
Comment (pharingee) [inline package.json]: Is there a reason we aren't using a range for `react-router-dom`?

File: package.json (hunk 1 above)
Comment (pharingee) [reply package.json:172]: Nothing seems to be broken on my check

File: package.json (hunk 2 above)
Comment (thekp) [reply package.json]: That was my mistake, I was testing something with `react-router-dom` and forgot to remove my change 😅

File: package.json (hunk 1 above)
Comment (thekp) [reply package.json:172]: https://www.chromaticqa.com/build?appId=5d28eb3fe163f6002046d6fa&number=4929

looks like it still works from the build here as well ^^

**Review by pharingee — APPROVED** [review]
LGTM

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**